x-pack/test/functional/services/ml/data_frame_analytics_table.ts

```diff
@@ -88,6 +88,12 @@ export function MachineLearningDataFrameAnalyticsTableProvider({ getService }: F
       await testSubjects.existOrFail('mlAnalyticsJobViewButton');
     }
 
+    public async openEditFlyout(analyticsId: string) {
+      await this.openRowActions(analyticsId);
+      await testSubjects.click('mlAnalyticsJobEditButton');
+      await testSubjects.existOrFail('analyticsEditFlyout', { timeout: 5000 });
```

Comment:
Nit: we try to prefix all of our test subjects with `ml`, so this one could be renamed.

Reply:
Updated in 58d750b4268a3b164eda8f2b8cd3ab05160f9379